src/components/Radio/Radio.tsx

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import { createUseStyles } from 'react-jss';
+import classNames from 'classnames';
+import Colors from './../Colors';
+
+export type RadioProps = {
+  className?: string;
+  choiceClassName?: string;
+  name: string;
+	choices?: Choice[];
+	selected?: string;
+	onSelect: (choice: string, event: React.MouseEvent<HTMLInputElement>) => void;
+};
+
+export type Choice = {
+  text: string,
```

Comment:
I should have put label , more understandable for a radio button, but text is ok too